pkg/controller/execution_job.go

```diff
@@ -191,6 +220,138 @@ func (e *ExecutionJobController) syncHandler(event Event) error {
 	return nil
 }
 
+func evalgetResult(output string, vars []interface{}) ([]parser.Var, string, error) {
+
+	result := make([]parser.Var, 0, len(vars))
+	var parentJobName string
+	for _, vs := range vars {
+		v := vs.([]interface{})
+
+		if item, ok := v[0].(string); ok && item == "get_result" {
+
+			parentJobName := v[1].(string)
+			sep := v[2].(string)
+
+			if sep == "" {
+				temp := []interface{}{output}
+				result = append(result, temp)
+			} else {
+				var strslice []interface{}
+				for _, str := range strings.Split(output, sep) {
+					if str != "" {
+						strslice = append(strslice, str)
+					}
+				}
+				temp := []interface{}{strslice}
+				result = append(result, temp)
+			}
+			glog.Infof("In evalgetResult jobName: %s sep:%s", parentJobName, sep)
+		}
+	}
+	glog.Infof("In evalgetResult result: %v", result)
+	return result, parentJobName, nil
+}
+func ConvertVars(vars []interface{}) []parser.Var {
+	result := make([]parser.Var, 0, len(vars))
+
+	for _, v := range vars {
+		res := make([]interface{}, 0)
+		res = append(res, v)
+		result = append(result, res)
+	}
+	return result
+}
+
+func (e *ExecutionJobController) createDynamicJob(child *graph.Vertex, jobresult string, graph1 *graph.Graph, key string) error {
```

Comment:
createDynamicJob(child *graph.Vertex, jobResult string, graph *graph.Graph, key string)
```suggestion
func (e *ExecutionJobController) createDynamicJob(child *graph.Vertex, jobResult string, graph *graph.Graph, key string) error {
```